### ISSUE TITLE
Rust connectors

### ISSUE THREAD

**Issue by davidjeba** [yd-dhahn/rust-crm-sync#1]
Can you build identity provider system on Rust with firestore/surrealdb backend?

**Comment by dahateb** [comment]
Is this related to the project or something else?

**Comment by davidjeba** [comment]
yes, i have data stored in Firestore which has to be insync with SurrealDB database
if a rust based middleware can do the realtime sync between two database systems it would be great.
I can buy the script via https://extensions.dev/extensions?q=&firebase=datastore
